### DISCUSSION TITLE
20240111  Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by cgbosse** [nautilus-cyberneering/.github#77] [Meetings]
Time: 8:30 GMT+0  Canary Islands Time (9:30 in Vaduz)

Location:
<https://meeting.hodl10.io/ConsistentClassroomsMergeSideways>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- CGarnham(@da2ce7)  - x
- JCelano(@josecelano) - x
- CBosse(@cgbosse) - x
- MMelchor(@mario-nt) - x
- GByrne(@grmbyrn) - x
- MVanDijke(@WarmBeer) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by cgbosse** [comment]
I have created a project for Torrust, with several views.

https://github.com/orgs/torrust/projects/10/views/1

The purpose is for the team to:
- Know who is working on what.
- Requires help.
- What is up for review.
- Which issues may be forgotten.
- What maintenance needs to be done so that everyone chips in here who is working on Torrust.
- See which issues are not properly labeled.
- What feature requests exist.
- What are all the enhancements that could be shown on a roadmap view which we can develop.
- If there are any security problems.
- What bugs exist.
- Who has not assigned himself to the issues.
- Properly arrange the WIP issues and prs.
-

**Comment by mario-nt** [comment]
### GUI

- https://github.com/torrust/torrust-index-gui/issues/421
- https://github.com/torrust/torrust-index-gui/issues/401
- https://github.com/torrust/torrust-index-gui/issues/393


### Index



### Discussions

-  Read https://github.com/torrust/torrust-index/discussions/428

### Training/Research

-  Research on https://github.com/torrust/torrust-index-gui/issues/417
-  Research on https://github.com/squaresapp/rawjs
- CSS W3shools tutorials

**Comment by josecelano** [comment]
### Retro

#### Torrust Tracker

- PR review: [#[563](https://github.com/torrust/torrust-tracker/pull/563)] [Cleanup Announce Response](https://github.com/torrust/torrust-tracker/pull/563)

- Working on issue: **WIP** [#[582](https://github.com/torrust/torrust-tracker/issues/582)] [Error updating the tracker to the latest docker image in the live demo](https://github.com/torrust/torrust-tracker/issues/582). Should I continue? It seems @da2ce7 is [working on a refactor that does not have that problem](https://github.com/torrust/torrust-tracker/pull/594#issuecomment-1886132529).

#### Torrust Index

- New discussion: [#[428](https://github.com/torrust/torrust-index/discussions/428)] [Import tracker statistics in batches](https://github.com/torrust/torrust-index/discussions/428)
- New issue: [#[427](https://github.com/torrust/torrust-index/issues/427)] [Images proxy does not work with simple reverse proxy configuration in Nginx](https://github.com/torrust/torrust-index/issues/427)

### Discussion

- Add E2E test to tracker using a shared dockerized instance like in the Index? To about problems like: "[Error updating the tracker to the latest docker image in the live demo](https://github.com/torrust/torrust-tracker/issues/582)"

**Comment by grmbyrn** [comment]
## Torrust Website
* Modal in header to link to most recent blog post
* Strap at the top of page to link to GitHub
* Components to wrap around blog posts to make table of contents responsive
* Share buttons

Revision/Research
* Making [BrainScape](https://www.brainscape.com/p/59GDY-LH-CYDRB) flashcards for Git, Svelte, and Python.
* Docker

**Comment by mickvandijke** [comment]
## Torrust Tracker

I have been working on adding a configurable memory limit to the torrent repository: https://github.com/torrust/torrust-tracker/issues/567
